config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-encryption-algorithm.yaml

```diff
@@ -0,0 +1,46 @@
+# AUTOGENERATED by https://git.k8s.io/kubeadm/kinder/ci/tools/update-workflows
+periodics:
+
+- name: ci-kubernetes-e2e-kubeadm-kinder-encryption-algorithm-latest
+  cluster: k8s-infra-prow-build
```

Comment:
todo: later, can we use the eks cluster for some of these?

Reply:
i though the jobs were already migrated.
adding the update as part of this PR in a commit the generator is updated in the k/kubeadm PR too.

Reply:
they are migrated already, it's not high priority, but we want to load balance some to the AWS budget where we can, versus a lot of the "real" cluster e2e has to run on the GCP budget both for the test cluster and for the prow pod.

Reply:
i forgot that k8s-infra-prow-build is already community infra.
updated the PR again. we can try all kubeadm jobs on EKS for a while and see how it goes.